reference-implementation/to-upstream-wpts/resources/test-utils.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+self.getterRejects = (t, obj, getterName, target) => {
```

Comment:
I would prefer assertGetterRejects(), to make it clear that this is an assertion.

I thought about assert_getter_rejects(), but then people might expect to find the definition in testharness.js.

I realise that testharness's "promise_rejects" doesn't include "assert", but I find that a bit counter-intuitive to be honest.